content/join/index.md

```diff
@@ -7,6 +7,22 @@ menu:
     weight: 1
 ---
 
+# Contributing to the OPI Project
+
+Please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdYilTsg74EZCh2_5G6dPtNd0z3EQv4Pc63brUUpM1XMQkzMg/viewform?usp=sf_link)
+if you are interested in contributing to the Open Programmable Infrastructure
+Project. Once you have filled out the participation form you will be added to
+the contact sheet and have access to edit it. You can then join any desired
+subgroup by adding an `x` to the appropriate box. Also, it will help to email
+the group leader to make sure you are included in invites.
+
+## Request New Member Orientation
```

Comment:
Can we remove this section ( the new member part ) and just have the one google form for contributing?